library/src/test/java/bio/terra/pfb/PfbReaderTest.java

```diff
@@ -38,6 +45,38 @@ void showSchemaTest(String fileName) throws IOException {
     }
   }
 
+  // this test validates behavior of the PfbReader.getPfbSchema() method. This method is
+  // used internally by PfbReader.showSchema, which is thoroughly tested above; therefore,
```

Comment:
```suggestion
  // used internally by PfbReader.showSchema, which is thoroughly tested by {@link #showsSchemaTest()} therefore,
```

(Javadoc linkiness completely optional)

Reply:
Also, the reason I wanted clarification, is the code above this looks like a lot less test code than what this provides and claims is cursory only =)

Maybe the thing that should be linked is `assertJavaPfbIsPyPFB`?

Reply:
well, `showsSchemaTest` loops over the test files and calls `assertJavaPfbIsPyPFB` for each one, so it feels pretty much the same?